spec/controllers/api/v1/reviews_controller_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ReviewsController, type: :controller do
+  let!(:user1) {User.create(
+    email: "yo@yo.com",
+    password: "password1",
+    role: "member"
+    )}
+
+  let!(:user2) {User.create(
+    email: "no@yo.com",
+    password: "password2",
+    role: "member"
+      )}
+
+  let!(:album1) {Album.create(
+    album: "test album 1",
+    art: "https://vignette.wikia.nocookie.net/selenagomez/images/4/46/Rare_Album_Cover.jpg/revision/latest?cb=20191212172712",
+    artist: "Selena Gomez",
+    year: 2020,
+    genre: "Pop",
+    user: user2
+    )}
+
```

Comment:
closing `)}` on lines 24 and 23 should line up with the declaration of your `let!`